Be able to use fixed tag (#8713)
camptocamp/c2cgeoportal: doc/integrator/security.rst

.. include:: https.rst
.. include:: reset_password.rst

Security update
---------------

To be sure that we regularly get the security updates, every night the GeoMapFish Docker images are rebuild.
And every time we do a build we pull the new images to use them.

> For project on Kubernetes we also rebuild them every day.

Reviewer: ```suggestion
For project on Kubernetes we also deploy fresh builded images every day.
```

> This is good for security but with that we can't guarantee that the result of a new build work as the

Reviewer: ```suggestion
This is good for security but with that we can't guarantee that the result of a new build works exactly as the
```

previous one.

> Too guarantee the incident on the production, if we are on Kubernetes if the new image will not start he

Reviewer: ```suggestion
To avoid incidents on production service, Kubernetes will publish a service which start correctly.

> will not use it.  If we are on docker-compose we have tow chose. The best is to build the image only on

Reviewer: ```suggestion
will not use it. If we are on docker-compose we have to choose ourself. The best is to build the image only on
```

integration when the result is correct we push it on a repository, and on production we will use the

> images from the repository. The other solution is to use fixed tag of an image, this imply that we

Reviewer: ```suggestion
images from the repository. The other solution is to use fixed tag for the base image, this imply that we
```
